src/modules/launcher/Plugins/Microsoft.Plugin.WindowWalker/Components/Window.cs

```diff
@@ -51,11 +46,16 @@ public string Title
         {
             get
             {
-                int sizeOfTitle = InteropAndHelpers.GetWindowTextLength(hwnd);
+                int sizeOfTitle = NativeMethods.GetWindowTextLength(hwnd);
                 if (sizeOfTitle++ > 0)
                 {
                     StringBuilder titleBuffer = new StringBuilder(sizeOfTitle);
-                    InteropAndHelpers.GetWindowText(hwnd, titleBuffer, sizeOfTitle);
+                    var numCharachtersWritten = NativeMethods.GetWindowText(hwnd, titleBuffer, sizeOfTitle);
```

Comment:
[nit] typo: Character.

Reply:
arghh.  I can't tell you how many times I've mistyped the word 'character'.  Somewhere along the line I developed some bad muscle memory.  Thanks for catching this.   I've updated this.